bin/apisix

```diff
@@ -103,8 +103,10 @@ worker_shutdown_timeout 3;
 {% if stream_proxy then %}
 stream {
     lua_package_path  "$prefix/deps/share/lua/5.1/?.lua;/usr/share/lua/5.1/apisix/lua/?.lua;]=]
+                      .. [=[/usr/local/apisix/deps/share/lua/5.1/apisix/lua/?.lua;]=]
                       .. [=[{*apisix_lua_home*}/lua/?.lua;;{*lua_path*};";
     lua_package_cpath "$prefix/deps/lib64/lua/5.1/?.so;]=]
+                      .. [=[/usr/local/apisix/deps/lib/lua/5.1/?.so;]=]
```

Comment:
I do not think this line is useful. We should drop it.

Reply:
understood